SECURITY.md

```diff
@@ -0,0 +1,174 @@
+# Security Policy
+
+This document includes information about the vulnerability reporting, patch,
+release, and disclosure processes, as well as general security posture.
+
+<!-- markdown-toc --bullets="-" -i SECURITY.md -->
+<!-- toc -->
+
+- [Supported Versions](#supported-versions)
+- [Reporting a Vulnerability](#reporting-a-vulnerability)
+  - [When Should I Report a Vulnerability?](#when-should-i-report-a-vulnerability)
+  - [When Should I NOT Report a Vulnerability?](#when-should-i-not-report-a-vulnerability)
+  - [Vulnerability Response](#vulnerability-response)
+- [Security Release & Disclosure Process](#security-release--disclosure-process)
+  - [Private Disclosure](#private-disclosure)
+  - [Public Disclosure](#public-disclosure)
+  - [Security Releases](#security-releases)
+  - [Severity](#severity)
+- [Security Posture](#security-posture)
+- [Security Team](#security-team)
+- [Security Policy Updates](#security-policy-updates)
+
+<!-- tocstop -->
+
+## Supported Versions
+
+The following versions are currently supported and receive security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.2.x   | :white_check_mark: |
+| 1.1.x   | :x:                |
+| 0.0.x   | :x:                |
+
+## Reporting a Vulnerability
+
+We're extremely grateful for security researchers and users that report
+vulnerabilities to us. All reports are thoroughly investigated by the project
+[security team](#security-team).
+
+Vulnerabilities are reported privately via GitHub's
+[Security Advisories](https://docs.github.com/en/code-security/security-advisories)
+feature. Please use the following link to submit your vulnerability:
+[Report a vulnerability](https://github.com/slsa-framework/slsa-github-generator/security/advisories/new)
+
+Please see
+[Privately reporting a security vulnerability](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability#privately-reporting-a-security-vulnerability)
+for more information on how to submit a vulnerability using GitHub's interface.
+
+### When Should I Report a Vulnerability?
+
+- You think you discovered a potential security vulnerability in slsa-github-generator
+- You are unsure how a vulnerability affects slsa-github-generator
+- You think you discovered a vulnerability in another project that slsa-github-generator depends on
+  - For projects with their own vulnerability reporting and disclosure process, please report it directly there
+
+### When Should I NOT Report a Vulnerability?
+
+- You need help tuning GitHub Actions for security
+- You need help applying security related updates
+- Your issue is not security related
+
+### Vulnerability Response
+
+Each report is acknowledged and analyzed by the [Security Team](#security-team)
+within 14 days. This will set off the
```

Comment:
14 days seems like a long time to just acknowledge a report but that was the limit for the "initial response time" in the OpenSSF best practices (assuming "initial response time" meant acknowledgement of the vulnerability report and not public disclosure) and I didn't want to necessarily promise more than that.

@laurentsimon @asraa @joshuagl Thoughts?

Reply:
I'm fine with 14 days to start with.

Reply:
Agreed, makes sense to start with a larger value and reduce over time as we gain confidence.